src/export/ExportWavPack.cpp

```diff
@@ -0,0 +1,489 @@
+   XO("High Quality (Slow)") ,
+   XO("Very High Quality (Slowest)") ,
+};
+
+const std::vector< int > ExportQualityValues{
+   0,
+   1,
+   2,
+};
+
+namespace {
+
+const TranslatableStrings ExportBitDepthNames{
+   XO("16 bit") ,
+   XO("24 bit") ,
+   XO("32 bit float ") ,
+};
+
+const std::vector< int > ExportBitDepthValues{
+   16,
+   24,
+   32,
+};
+
+IntSetting QualitySetting{ L"/FileFormats/WavPackEncodeQuality", 1 };
+IntSetting BitrateSetting{ L"/FileFormats/WavPackBitrate", 160 };
+IntSetting BitDepthSetting{ L"/FileFormats/WavPackBitDepth", 16 };
+
+BoolSetting HybridModeSetting{ L"/FileFormats/WavPackHybridMode", false };
+BoolSetting CreateCorrectionFileSetting{ L"/FileFormats/WavPackCreateCorrectionFile", false };
+
+/* 
+Copied from ExportMP2.cpp by
+   Joshua Haberman
+   Markus Meyer
+*/
+
+// i18n-hint kbps abbreviates "thousands of bits per second"
+inline TranslatableString n_kbps( int n ) { return XO("%d kbps").Format( n ); }
+
+const TranslatableStrings BitRateNames {
+   n_kbps(16),
+   n_kbps(24),
+   n_kbps(32),
+   n_kbps(40),
+   n_kbps(48),
+   n_kbps(56),
+   n_kbps(64),
+   n_kbps(80),
+   n_kbps(96),
+   n_kbps(112),
+   n_kbps(128),
+   n_kbps(160),
+   n_kbps(192),
+   n_kbps(224),
+   n_kbps(256),
+   n_kbps(320),
+   n_kbps(384),
+};
+
+const std::vector< int > BitRateValues {
+   16,
+   24,
+   32,
+   40,
+   48,
+   56,
+   64,
+   80,
+   96,
+   112,
+   128,
+   160,
+   192,
+   224,
+   256,
+   320,
+   384,
+};
+
+}
+
+void ExportWavPackOptions::PopulateOrExchange(ShuttleGui & S)
+{
+   bool hybridMode = HybridModeSetting.Read();
+
+   S.StartVerticalLay();
+   {
+      S.StartHorizontalLay(wxEXPAND);
+      {
+         S.SetSizerProportion(1);
+         S.StartMultiColumn(2, wxCENTER);
+         {
+            S.TieNumberAsChoice(
+               XXO("Quality"),
+               QualitySetting,
+               ExportQualityNames,
+               &ExportQualityValues
+            );
+
+            S.TieNumberAsChoice(
+               XXO("Bit Depth"),
+               BitDepthSetting,
+               ExportBitDepthNames,
+               &ExportBitDepthValues
+            );
+
+            S.Id(ID_HYBRID_MODE).TieCheckBox( XXO("Hybrid Mode"), HybridModeSetting);
+            mCreateCorrectionFile = S.Disable(!hybridMode).TieCheckBox( XXO("Create Correction(.wvc) File"), CreateCorrectionFileSetting);
+
+            mBitRate = S.Disable(!hybridMode).TieNumberAsChoice(
+               XXO("Bit Rate:"),
+               BitrateSetting,
+               BitRateNames,
+               &BitRateValues
+            );
+         }
+         S.EndMultiColumn();
+      }
+      S.EndHorizontalLay();
+   }
+   S.EndVerticalLay();
+}
+
+bool ExportWavPackOptions::TransferDataToWindow()
+{
+   return true;
+}
+
+bool ExportWavPackOptions::TransferDataFromWindow()
+{
+   ShuttleGui S(this, eIsSavingToPrefs);
+   PopulateOrExchange(S);
+
+   CreateCorrectionFileSetting.Write( mCreateCorrectionFile->GetValue() );
```

Comment:
Hmm, I'm not sure why this is needed. 

And I think we need to flush settings here